en/mulesoft/modules/ROOT/pages/ms_composer_flows.adoc

```diff
@@ -131,6 +132,22 @@ To repair a connection:
 . Click the change icon in the trigger or action that requires repair, and then click *Change Connection*.
 . Click *Repair*.
 
+== Delete a Connection
+
+If you no longer need a specific OAuth connection, you can delete it.
+
+When you attempt to delete a connection, Composer ensures that deleting the connection will not affect other flows.
+
+To delete a connection:
+
+. <<create-a-new-flow,Create a new flow.>>
+. In the *What Should Start This Flow?* window, click the change icon. +
```

Comment:
"What Should Start This Flow" or "Add Step"

Reply:
Also, the gear icon appears after you've selected the connector/system of choice.